libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SalesforceSDKCoreDefines.h

```diff
@@ -38,40 +38,6 @@ enum {
     kSalesforceSDKManagerErrorInvalidLaunchParameters
 };
 
-// Launch actions taken
-typedef NS_OPTIONS(NSInteger, SFSDKLaunchAction) {
```

Comment:
I missed these when adding deprecation markers, but everything that uses them was deprecated